src/github-issues-prs.ts

```diff
@@ -394,7 +398,8 @@ export class GitHubIssuesPrsProvider implements TreeDataProvider<TreeItem> {
 			try {
 				const { stdout } = await exec('git remote -v', { cwd: folder.uri.fsPath });
 				for (const url of new Set(allMatches(/^[^\s]+\s+([^\s]+)/gm, stdout, 1))) {
-					const m = /[^\s]*github\.com[/:]([^/]+)\/([^ ]+)[^\s]*/.exec(url);
+					const m = new RegExp(`[^\s]*${this.getURL().replace(/\./g, '\\.')}[/:]([^/]+)\/([^ ]+)[^\s]*`).exec(url);
```

Comment:
I think the `\s` still need escaping as `\\s`, don't they? That's because it is in a string and not in a regex anymore.

Reply:
I missed it. Thank you